ktor-features/ktor-serialization/jvm/test/io/ktor/tests/serialization/SerializationTest.kt

```diff
@@ -61,6 +61,81 @@ class SerializationTest {
         }
     }
 
+    @Test
+    fun testEntityListReceive(): Unit = withTestApplication {
```

Comment:
should we add same tests for send?

Reply:
It won't work anyway since there is no way to pass KType when responding with a message. The only possible solution is to create serializer similar to the client's implementation.

Reply:
Ok, I've added required support for sending collections with tests